source/solvers/heat_transfer.cc

```diff
@@ -969,7 +1019,780 @@ HeatTransfer<dim>::solve_linear_system(const bool initial_step,
   newton_update = completely_distributed_solution;
 }
 
+template <int dim>
+void
+HeatTransfer<dim>::postprocess_temperature_statistics(
+  const bool                       gather_vof,
+  const Parameters::FluidIndicator monitored_fluid,
+  const std::string                domain_name)
+{
+  const unsigned int n_q_points       = this->cell_quadrature->size();
+  const MPI_Comm     mpi_communicator = this->dof_handler.get_communicator();
+
+  // Initialize heat transfer information
+  std::vector<double> local_temperature_values(n_q_points);
+  FEValues<dim>       fe_values_ht(*this->temperature_mapping,
+                             *this->fe,
+                             *this->cell_quadrature,
+                             update_values | update_JxW_values);
+
+  // Initialize VOF information
+  const DoFHandler<dim> *        dof_handler_vof;
+  std::shared_ptr<FEValues<dim>> fe_values_vof;
+  std::vector<double>            phase_values(n_q_points);
+
+  if (gather_vof)
+    {
+      dof_handler_vof = this->multiphysics->get_dof_handler(PhysicsID::VOF);
+      fe_values_vof =
+        std::make_shared<FEValues<dim>>(*this->temperature_mapping,
+                                        dof_handler_vof->get_fe(),
+                                        *this->cell_quadrature,
+                                        update_values);
+    }
+
+  // Other initializations
+  double phase_coefficient(0.);
+  bool   take_min_max(false);
```

Comment:
Can you add a comment that describes what is the take_min_max variable?

Reply:
Thanks to your comment, I have been able to simplify that, thank you :)